### PR TITLE
ASAP-Alchemy: Allow Postera molecule sets as input

### PR DESCRIPTION
**Pull request by jthorton**
## Description
This PR extends the ASAP-Alchemy prep CLI to take in postera molecule sets as input.

## Todos
Notable points that this PR has either accomplished or will accomplish.
- [X] tests

## Questions
- [ ] Question 1 ..

## Status
- [x] Ready to go

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT license as defined in our [LICENSE](https://github.com/choderalab/asapdiscovery/blob/main/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

### CODE REVIEW

File: asapdiscovery-alchemy/asapdiscovery/alchemy/cli/utils.py [hunk 1]
```diff
@@ -2,8 +2,26 @@
...
 def print_header(console: "rich.Console"):
-    "Print an ASAP-Alchemy header"
+    """Print an ASAP-Alchemy header"""
 
     console.line()
     console.rule("ASAP-Alchemy")
     console.line()
+
+
+def pull_from_postera(molecule_set_name: str):
```
Comment (JenkeScheen) [inline asapdiscovery-alchemy/asapdiscovery/alchemy/cli/utils.py:12]: perhaps this should be defined in `asapdiscovery-data/postera` instead?

Comment (jthorton) [reply asapdiscovery-alchemy/asapdiscovery/alchemy/cli/utils.py:12]: I would like to keep it separate if possible as it allows us to tuck the imports so you don't end up importing a load of redundant packages if you don't want to use postera.

**Review by JenkeScheen — APPROVED** [review]
LGTM!

**Review by hmacdope — APPROVED** [review]
LGTM great work

**Pull request merged**